Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM runatlantis/atlantis:latest
+
+ENV TERRAGRUNT_VERSION=v0.21.7
+
+RUN curl -s -Lo terragrunt https://github.com/gruntwork-io/terragrunt/releases/download/${TERRAGRUNT_VERSION}/terragrunt_linux_amd64 && \
+    chmod +x terragrunt && \
+    mv terragrunt /usr/local/bin
+RUN apk add unzip make curl
```

Comment:
I would also go for `RUN set -eux && ...` for good practice.

Reply:
Why would we need this in a `Dockerfile`?

Reply:
`-x` to see the output during build and `-eu` is just best practice in shell code (which it basically is)

Reply:
This means we would need to prefix each `RUN` directive with `set -eux`... I think this is an overkill since `RUN` directives are not complex scripts. But I see we have been using this previously: https://github.com/Flaconi/docker-nginx-base/blob/master/Dockerfile#L6-L7

If this is the standard lets use and move forward:

```bash
RUN set -eux \
	&& apk add unzip make curl
```